x/wasm/keeper/keeper_test.go

```diff
@@ -1501,33 +1392,74 @@ func TestClearContractAdminInactiveContract(t *testing.T) {
 	wasmCode, err := ioutil.ReadFile("./testdata/hackatom.wasm")
 	require.NoError(t, err)
 
-	originalContractID, err := keeper.Create(ctx, creator, wasmCode, nil)
+	contractID, err := keeper.Create(ctx, creator, wasmCode, nil)
 	require.NoError(t, err)
 
-	_, _, anyAddr := keyPubAddr()
+	_, _, bob := keyPubAddr()
 	initMsg := HackatomExampleInitMsg{
 		Verifier:    fred,
-		Beneficiary: anyAddr,
+		Beneficiary: bob,
 	}
 	initMsgBz, err := json.Marshal(initMsg)
 	require.NoError(t, err)
 
-	addr, _, err := keeper.Instantiate(ctx, originalContractID, creator, fred, initMsgBz, "demo contract", nil)
+	addr, _, err := keeper.Instantiate(ctx, contractID, creator, "", initMsgBz, "demo contract 3", deposit)
 	require.NoError(t, err)
+	require.Equal(t, "link14hj2tavq8fpesdwxxcu44rty3hh90vhud63e6j", addr.String())
 
-	params := wasmKeeper.GetParams(ctx)
-	params.ContractStatusAccess = types.AccessTypeOnlyAddress.With(creator)
-	wasmKeeper.setParams(ctx, params)
+	// execute inactive contract through manual
 	err = keeper.UpdateContractStatus(ctx, addr, creator, types.ContractStatusInactive)
+
+	// Contract status can only be changed through a gov proposal.
+	require.True(t, sdkerrors.ErrUnauthorized.Is(err), "expected %v but got %v", sdkerrors.ErrUnauthorized, err)
+}
+
+func TestMigrateInactiveContract(t *testing.T) {
```

Comment:
ditto.

Reply:
These tests below test whether other features are normally blocked after disabling them through the inactive gov proposal.
 I've edited it to have a more precise meaning. 
thks!